lgc/builder/YCbCrAddressHandler.cpp

```diff
@@ -47,19 +47,37 @@ void YCbCrAddressHandler::genBaseAddress(unsigned planeCount) {
   // For YCbCr, the possible plane counts are among 1 and 3.
   assert(planeCount > 0 && planeCount < 4);
 
-  // PlaneBaseAddresses[0] is the same as original base addressed as passed in SRD
-  m_planeBaseAddresses.push_back(m_regHandler->getReg(SqRsrcRegs::BaseAddress));
+  // PlaneBaseAddresses[0] is the same as original base addressed as passed in SRD, and is already pipeBankXored
+  Value *virtualAddrPlane0 = m_regHandler->getReg(SqRsrcRegs::BaseAddress);
+  m_planeBaseAddresses.push_back(virtualAddrPlane0);
 
+  // Jude if current swizzle mode is ADDR_SW_64KB_R_X
+  Value *isSW_64KB_R_X_Mode = m_builder->CreateICmpEQ(m_swizzleMode, m_builder->getInt32(27));
+
+  // Calculate the pipeBankXor for second plane
+  Value *pipeBankXor1 =
+      m_builder->CreateSelect(isSW_64KB_R_X_Mode, m_builder->getInt32(4 << 5), m_builder->getInt32(0 << 5));
```

Comment:
`0 << 5` is 0. Is this intentional?

Reply:
Yes, this is intentional :)

Reply:
Can you add a comment explaining why why the `5` is important here?

Reply:
Updated with more details in code.

Reply:
Thanks